File: nvim/lua/specto/examples/spec.js
// example of a jest test file
xdescribe('foo', () => {
  it('bar', () => {
    expect(1).toBe(1)
  })
})

describe.only('foobar', () => {
  it.only('barfoo', () => {
    expect(1).toBe(1)
  })
})

File: nvim/lua/specto/examples/spec.rb
# frozen_string_literal: true

# example of a jest test file
xdescribe('foo') do
  xit('bar') do
    expect(1).toBe(1)
  end
end

xcontext('foobar') do
  xscenario('barfoo') do
    expect(1).toBe(1)
  end
end
